.circleci/config.yml

```diff
@@ -239,7 +238,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
```

Comment:
Non-blocker: I would probably just leave `master` in there too

Reply:
> Fine by me as long as we understand why the UAT deploy failed. Is it related to trying to do both staging and UAT with this branch?

It seemed to have created pods fine with all the pods in the Running state but the job failed. Rerunning it again seemed to have made it pass, perhaps it just timed out